kloppy/domain/models/pitch.py

```diff
@@ -699,3 +699,24 @@ class WyscoutPitchDimensions(PitchDimensions):
     corner_radius: float = 0.97  # inferred
     penalty_spot_distance: float = 10.0
     penalty_arc_radius: float = 7.74  # inferred
+
+
+@dataclass
+class ImpectPitchDimensions(PitchDimensions):
```

Comment:
These are `MetricPitchDimensions`. It is not needed to create a separate object.